css/src/core/bare-elements.scss

```diff
@@ -32,3 +32,12 @@ fieldset {
 	min-width: 0; // as a replaced element min-width:0 is required to enable it to work in responsive layouts
 	border: none;
 }
+
+::target-text {
```

Comment:
Looks like it's not supported by Firefox and Safari. Is there a fallback for non-supported browsers? 
![image](https://user-images.githubusercontent.com/94572161/231558583-7939f3db-5eb6-4d70-8922-aac640b40b33.png)